pom.xml

```diff
@@ -159,6 +159,7 @@ flexible messaging model and an intuitive client API.</description>
     <scala.binary.version>2.13</scala.binary.version>
     <scala-library.version>2.13.6</scala-library.version>
     <debezium.version>1.7.2.Final</debezium.version>
+    <debezium.postgresql.version>42.2.25</debezium.postgresql.version>
```

Comment:
Does this have to be configured separately?

Reply:
> Note: the version is not shared on purpose because we should leave the driver dependencies separated since the two connectors are used in different ways. (For example, when we'll upgrade Debezium to 1.8.x we'll need to remove the override and keep the 42.3.x version)